_layouts/base-styleguide.html

```diff
@@ -38,7 +38,7 @@
 {% endif %}
 
   <link rel="canonical" href="{{ site.url }}{{ site.baseurl }}{{ page.url }}" />
-
+  <script async type="text/javascript" id="_fed_an_ua_tag" src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency=DOC&subagency=CEN"></script>
```

Comment:
I think you can remove this change since we should only need to add the DAP script inside of `head` tags.

Reply:
@curt-mitch-census Will do — thanks for catching that! 🙌🏾